### ISSUE TITLE
Bug Sync Settings Bundle Plugin

### ISSUE THREAD

**Issue by Cleboost** [pandier/intellij-discord-rp#65]
### What happened?

I can setup my status but when I restart my IDE, the last change is not here...
This does not work for me unless Settings Sync is enabled.

https://nishbdorg-my.sharepoint.com/:v:/g/personal/100265_365net_co/EVPCbv9jATxDjNng4ecFu-sBUYmjEthhII9MHQRsZklBkA?e=bSyISy

### Expected behaviour

I found this : https://plugins.jetbrains.com/docs/intellij/persisting-state-of-components.html#settings-sync-plugin
I don't know if you enable it.

![image](https://github.com/pandier/intellij-discord-rp/assets/61158869/6f3cfd2d-55f8-4d80-be8a-63a18449076f)

### What version of the plugin are you using?

1.2

### What IDE and version are you using?

WebStorm 2024.1.2

**Comment by pandier** [comment]
Can confirm this happens. Seems to be an issue with how IntelliJ api encodes settings, because if you put an emoji directly into the settings file, it works. This is not handled by me. I'll ask on the JetBrains Platform Slack.

**Comment by Cleboost** [comment]
So if i write my emoji in file it's work ? PS : where is the config file x)

**Comment by pandier** [comment]
Yes, it should work. In Windows the config file is located at `%APPDATA%\JetBrains\YOUR IDE\options\discordrp.xml`.

**Comment by Cleboost** [comment]
Thanks

**Comment by pandier** [comment]
Related JetBrains YouTrack issue: https://youtrack.jetbrains.com/issue/IJPL-38379/Paths-with-emoji-dont-get-saved-after-closing
